rack-box/http/user-data

```diff
@@ -27,8 +27,10 @@ autoinstall:
     - python3
     - python3-pip
     - strace
+    - sudo
     - swi-prolog
     - unzip
     - vim
   late-commands:
     - echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' > /target/etc/sudoers.d/ubuntu
+    - echo 'rackui ALL=(ALL) NOPASSWD:ALL' >> /target/etc/sudoers.d/ubuntu
```

Comment:
To answer your question whether we really need to modify user-data, the answer is that we used to build a virtual machine from an ISO for which we needed this user-data file.  However, the latest Ubuntu 20.04.5 point release ISO no longer boots and runs the install successfully, so I had to build the virtual machine from an existing virtual machine (a recent Ubuntu vagrant box) instead.  It still would be good to keep user-data around and up to date in case we need to build a virtual machine from an ISO again, so this change is good (although sudo is already installed by the ISO and doesn't need to be listed here).

I suggest changing the last line to create a different and new file to reduce conflict with install.sh:

`    - echo 'rackui ALL=(ALL) NOPASSWD:ALL' > /target/etc/sudoers.d/rackui`